### DISCUSSION TITLE
2023년 02월 06일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#17] [General]
# 2023년 02월 06일

[ 2023-02-06 ] [ 2023-02-07 ] [ 2023-02-08 ] [ 2023-02-09 ] [ 2023-02-10 ] [ 2023-02-12 ] [ 2023-02-15 ] [ 2023-02-16 ] [ 2023-02-17 ] [ 20…

https://cyma-s.github.io/2023-02-06/

**Comment by Choi-JJunho** [comment]
베로의 첫 우테코생활 기대가 되는데요?!
앞으로의 행보 잘 지켜보겠습니다👀
